src/gui/editorwidgets/qgsmultiedittoolbutton.cpp

```diff
@@ -0,0 +1,126 @@
+/***************************************************************************
+     qgsmultiedittoolbutton.cpp
+     --------------------------
+    Date                 : March 2016
+    Copyright            : (C) 2016 Nyall Dawson
+    Email                : nyall dot dawson at gmail.com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsmultiedittoolbutton.h"
+#include "qgsapplication.h"
+#include <QMenu>
+
+QIcon QgsMultiEditToolButton::sIconInitialState;
```

Comment:
And there is really no longer a reason for static QIcons since https://github.com/qgis/QGIS/commit/c9588cd56ea2de9aa004d975a19db3b6aec4516d